### DISCUSSION TITLE
Comments and remark42

### DISCUSSION THREAD

**Discussion by gleb-kun** [adityatelange/hugo-PaperMod#1262] [Q&A]
How to include remark42 comments? Where can you find information about this?

**Comment by c2tz** [comment]
Hello, in case the problem is still there I think you should consult the documentation here: [https://remark42.com/docs/getting-started/installation/](https://remark42.com/docs/getting-started/installation/)

Then just add some layout with the JS codes that are provided then the `<div id="remark42"></div>`.

Just to clarify, did you already consult the documentation? Why choose to use Remark42 over Giscus, for example? Are there any features you can't find elsewhere?

**Reply by gleb-kun** [reply]
I looked at the documentation. However, Jimmy's website states that the theme supports remark42 (https://stack.jimmycai.com/config/comments). I thought that some remark42 features were already integrated into the theme, as was done with Disqus (You just need to include comments in the yaml/toml config file). Or what is this support?
